input_samples/sample_jsons/run3_dilepton_skims/data_skims_hpg/DoubleMuon_B_2L_Loose_Skim.json

```diff
@@ -0,0 +1,23 @@
+{
+    "xsec": 1.0,
+    "year": "2022",
+    "treeName": "Events",
+    "era": "B",
+    "histAxisName": "",
+    "options": "",
+    "WCnames": [],
+    "files": [
+        "/cmsuf/data/store/user/t2/users/matthew.dittrich/skims/Run3_Data_2L_02162024_Loose/DoubleMuon_Run2022B-22Sep2023-v1_NANOAOD_Run3_Data_2L_02162024_Loose/output_7.root",
+        "/cmsuf/data/store/user/t2/users/matthew.dittrich/skims/Run3_Data_2L_02162024_Loose/DoubleMuon_Run2022B-22Sep2023-v1_NANOAOD_Run3_Data_2L_02162024_Loose/output_2.root",
```

Comment:
Any file that lives in an xrd accessible location (i.e. all of these) should have their paths start with `/store` in the config. This is ideal because it allows us to either access the file via xrd (by prepending a redirector name) or access the file locally (by prepending the local path, in this case `/cmsuf/data`).

So I think we should update all of these jsons so that the paths start with "/store".

Reply:
There is another directory /data_skims/ and /mc_skims/ that is accessible via xrootD. This directory was made in the case where xrootD may give issues. I can add a README to explain this and the contigency that it will only work on hipergator?